config/locales/de.default.yml

```diff
@@ -106,41 +106,41 @@ de:
       second: Sekunde
       year: Jahr
   errors:
-    format: '%{attribute} %{message}'
+    format: '%{attribute} %{message}.'
     form_invalid: 
-      one: "Achtung. In einem Feld ist ein Fehler aufgetreten. Korrigiere den Fehler um fortzufahren."
-      other: "Achtung. In %{count} Feldern sind Fehler aufgetreten. Korrigiere die Fehler um fortzufahren."
+      one: "Achtung. In einem Feld ist ein Fehler aufgetreten. Korrigiere den Fehler, um fortzufahren"
+      other: "Achtung. In %{count} Feldern sind Fehler aufgetreten. Korrigiere die Fehler, um fortzufahren"
     messages:
-      accepted: muss akzeptiert werden.
-      blank: 'Feld ist leer. Bitte fülle es aus, um fortfahren zu können.'
-      present: darf nicht ausgefüllt werden.
+      accepted: muss akzeptiert werde
+      blank: 'Feld ist leer. Bitte fülle es aus, um fortzufahren'
+      present: darf nicht ausgefüllt werden
       confirmation: stimmt nicht mit %{attribute} überein.
-      empty: Feld muss ausgefüllt werden.
-      equal_to: muss genau %{count} sein.
-      even: muss gerade sein.
-      exclusion: ist nicht verfügbar.
-      greater_than: muss größer als %{count} sein.
-      greater_than_or_equal_to: muss größer oder gleich %{count} sein.
-      inclusion: ist kein gültiger Wert.
-      invalid: ist nicht gültig.
-      less_than: muss kleiner als %{count} sein.
-      less_than_or_equal_to: muss kleiner oder gleich %{count} sein.
+      empty: 'Feld ist leer. Bitte fülle es aus, um fortzufahren'
+      equal_to: muss genau %{count} sein
+      even: muss gerade sein
+      exclusion: ist nicht verfügbar
+      greater_than: muss mindestens %{count} sein
+      greater_than_or_equal_to: darf höchstens %{count} sein
+      inclusion: ist kein gültiger Wert
+      invalid: ist nicht gültig
+      less_than: draf höchstens %{count} sein
+      less_than_or_equal_to: muss mindestens %{count} sein
```

Comment:
... why are the messages for <= and > identical? :sweat_smile: 
Same for >= and <.